files/plugins/glance_api_local_check.py

```diff
@@ -16,12 +16,9 @@
 
```

Comment:
I'd rather see this effort put into the monitorstack plugin manager found here https://github.com/openstack/monitorstack than for us to continue to carry these plugins.